omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -9,7 +9,7 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
 
     $scope.patientUuid = null;
     $scope.condition = null;
-
+    const inactiveStatus = 'INACTIVE';  
```

Comment:
Have you looked at our convention for constants? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions